modules/test/playwright/tests/message-boards-web/message-boards-web.spec.ts

```diff
@@ -178,3 +178,40 @@ test('LPD-37727 Change delta to a higher value when on last page', async ({
 		page.getByRole('link', {name: /Thread with headline #\d+/})
 	).toHaveCount(5);
 });
+
+test('LPD-39570 Change delta to a higher value when on last page', async ({
```

Comment:
@boton already did that in https://github.com/liferay-content-management/liferay-portal/pull/6014. To avoid massive conflicts, I pinky promise I'll do it in his PR (and fix the rebase conflicts myself).